py/selenium/webdriver/remote/errorhandler.py

```diff
@@ -208,12 +209,10 @@ def check_response(self, response: Dict[str, Any]) -> None:
         if message == "" and 'message' in value:
             message = value['message']
 
-        screen = None  # type: ignore[assignment]
         if 'screen' in value:
             screen = value['screen']
 
-        stacktrace = None
-        st_value = value.get('stackTrace') or value.get('stacktrace')
+        st_value = value.get('stacktrace')
```

Comment:
`stackTrace` was JWP stuff, I think safe to remove.  @titusfortner any idea if the `screen` code here can go? I only find references to it inside jwp specification docs

Reply:
Yes you can; it was one of the valid responses for jwp: https://www.selenium.dev/documentation/legacy/json_wire_protocol/#failed-commands
but is not one of the valid responses for w3c: https://w3c.github.io/webdriver/#errors